Terminal.Gui/Core/Window.cs

```diff
@@ -88,11 +88,18 @@ class ContentView : View {
 
 			public ContentView (Rect frame, Window instance) : base (frame)
 			{
-				this.instance = instance;
+				Initialize (instance);
 			}
 			public ContentView (Window instance) : base ()
+			{
+				Initialize (instance);
+			}
+
+			private void Initialize (Window instance)
```

Comment:
Can you please change this to `SetInitialProperties` so it's clear this is distinct from `Begin/EndInit`?

Reply:
@tig I don't mind to do that, but if you look for all files, including the `Window.cs` file, in the develop branch you will find `Initialize` method on all of them. I think you meaning that you want me to do that for the v2_developer branch, right?